package.json

```diff
@@ -23,6 +23,7 @@
   },
   "scripts": {
     "test": "mocha",
```

Comment:
1. Update this.
2. Add a `test-integration` script that runs the integration tests.